SP-2973: Updates for Q2 2026 (#63)
lsst/rtn-011: commissioning.tex

Initially, only small dithers were used, and no intra-night dithering was used. 
This configuration led to challenges with scattered light and the absence of intra-night dithers limited the ability to construct high-quality template images. 
Later in the campaign, both translational and rotational dithers were introduced within a night, while inter-night dithering was maintained. 
The optimal dither pattern for LSST DDF observations is the subject of ongoing investigations,  but will likely include both intra-night and inter-night dithering components.

\subsection{Start of Operations and the Start of the LSST}
\label{ssec:ops-lsst-start}

The \textit{Start of Rubin Operations} and the \textit{Start of the LSST} are two distinct milestones separated by the \textit{Rubin Operations Early Optimization Phase}.
Understanding the distinction between these two milestones, and what is required to transition from one to the other, is essential context for the Early Science Program.
 

> Following Rubin Construction completeness (\S\ref{ssec:commissioning-schedule}) on \cstrcompdate, Rubin Operations began the following day on \opsstartdate.

Reviewer: ```suggestion
Following Rubin Construction substantial completeness (\S\ref{ssec:commissioning-schedule}) on \cstrcompdate, Rubin Operations began the following day on \opsstartdate.
```

This milestone marked the formal handover of Rubin Observatory from the Construction project to the Operations team.
The \textit{Start of the LSST} is the subsequent milestone at which Rubin Observatory begins executing the Legacy Survey of Space and Time, as defined by the baseline survey strategy \citep{PSTN-056}.
 

> The period from the Start of Rubin Operations through the Start of the LSST is termed the \textit{Rubin Operations Early Optimization Phase} (\S~\ref{ssec:defn}).

Reviewer 2: ```suggestion
The period from the Start of Rubin Operations through the Start of the LSST is termed the \textit{Early Operations Optimization Phase} (\S~\ref{ssec:defn}).
```

During Construction Commissioning, the as-built system demonstrated that it is capable of meeting the LSST science requirements as defined in the Science Requirements Document \citedsp{LPM-17}.
However, demonstrating that the system \textit{can} achieve the required performance is not the same as demonstrating that it can do so in a sustained, reliable, and repeatable manner over the course of a 10-year survey.

> The goal of the Early Optimization Phase is to bridge this gap: to bring the system to a state of operational readiness in which it can execute the LSST within the specifications of the SRD, night after night, without interruption.

Reviewer: This is too strong. We may (will likely) start the LSST before performance aligned with SRD is fully realized. 

```suggestion
The goal of the Early Optimization Phase is to bridge this gap: to bring the system to a state of operational readiness that supports start of the LSST. It is expected that optimization will continue even after the start of the LSST.
```

> Once stable and repeatable performance at the required level has been achieved, the LSST can begin.

Reviewer: ```suggestion
%Once stable and repeatable performance at the required level has %been achieved, the LSST can begin.
```

 
Data collected during the Early Optimization Phase continue to provide the basis for early data products released to the community.
For example, early alerts based on templates built during the Commissioning period began streaming to community brokers on \rubinfirstalerts\footnote{\url{https://rubinobservatory.org/news/first-alerts}}, and the release of nightly Processed Visit Images (PVIs) is expected from \nightlypvis.
 
At the time of writing, Rubin Observatory remains in the Early Optimization Phase and the LSST has not yet begun.

> A start date for the LSST has not yet been determined but will be soon.

Reviewer 2: ```suggestion
A specific start date for the LSST has not yet been determined but will be soon; the date range in Table 7 gives the current best estimate for when that date will fall."
```

Author: There is no date range in Table 7 for the start of the LSST. Again, "soon" is vague undefined and IMHO weakens the statement  it by introducing vague optimism without adding helpful information.

Reviewer 2: I agree, let's work on replacing that "soon" with a real check-in date. I think @rblum5 has one in mind that we can use here. I don't think we should release a new version of RTN-011 until we have a date range for LSST start, so we need to estimate those dates with some urgency. I'll put it to the Start the LSST Board and also the Ops Exec that we should aim for our date ranges to represent 90% confidence intervals based on informed estimates.

Author: I made a suggestion above.  ->  A more concrete suggestion:
“A start date for the LSST has not yet been determined; this remains contingent on ongoing system optimization , and no firm timeline for setting the date is currently available."

> A start date for the LSST has not yet been determined.

Reviewer: ```suggestion
A start date for the LSST has not yet been determined but will be soon.
```

Author: I don't like this addition --  soon could be tomorrow or in 3 months. Its addition weakens the statement rather than strengthening it by introducing vague optimism without adding helpful information. “Soon” is undefined, not measurable, and doesn’t help anyone plan—particularly when the LSST is already late. I recommend removing it.

Author: A more concrete suggestion:
“A start date for the LSST has not yet been determined; this remains contingent on ongoing system optimization , and no firm timeline for setting the date is currently available."
